packages/js/src/Modules/Verto/BaseSession.ts

```diff
@@ -70,7 +70,7 @@ export default abstract class BaseSession {
   }
 
   get reconnectDelay() {
-    return randomInt(6, 2) * 1000;
+    return randomInt(2, 6) * 1000;
```

Comment:
This could lead to the same issues, won't it? Randomising doesn't guarantee that multiple clients won't connect at the same time.

Reply:
@StefanYohansson could you reply here, please?

Reply:
this random will decrease the pace. Of course, some can receive the same number, but the problem is not 5~10 connecting at the same time but 120~420 (current values of browsers reconnections).